export/export.go

```diff
@@ -0,0 +1,113 @@
+package export
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+	distrotypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	terra "github.com/terra-money/core/v2/app"
+)
+
+var IsAccountExportRunning = false
```

Comment:
Isn't this a local module variable? and I think `isAccountExportWorkerRunning` is more clearer

Reply:
+1